json-to-go.js

```diff
@@ -22,7 +22,7 @@ function jsonToGo(json, typename, flatten = true)
 
 	try
 	{
-		data = JSON.parse(json.replace(/\.0/g, ".1")); // hack that forces floats to stay as floats
+		data = JSON.parse(json.replace(/:(.*)\.0/g, ":$1.1")); // hack that forces floats to stay as floats
```

Comment:
Perhaps this should be...

```suggestion
		data = JSON.parse(json.replace(/:(\s*)\.0/g, ":$1.1")); // hack that forces floats to stay as floats
```

? Then again I haven't tested this, just trying to reduce false positives.

Reply:
Definitely shouldn't be `.*` here since that's greedy and will grab up to the _last_ occurence of `.0`. For non-pretty-printed input that will not work properly. @mholt's suggestion is what I'd use.

Reply:
`/:(\s*)\.0/g` will not match floats with hole numbers eg. 1.0 
we could change to `/:(\s*\d)\.0/g` which should match every float and ignore strings with .0 in them.

Compare https://regexr.com/4q6fj with https://regexr.com/4q6fa

Reply:
Yes, good point -- that looks better. Let's go with that for now.